gulpfile.js

```diff
@@ -57,8 +57,8 @@ function js (watch) {
   var shim = require('browserify-shim');
   var watchify = require('watchify');
 
-  var browserify_args = _.extend(watchify.args, {debug: true});
-  var bundler = browserify('./src/js/index.js', browserify_args);
+  var browserifyArgs = _.extend(watchify.args, {debug: true});
```

Comment:
`debug` generates source maps. Perhaps we can disable it in the production builds?